packages/jetpack-blocks/.eslintrc.js

```diff
@@ -1,23 +0,0 @@
-module.exports = {
```

Comment:
Should we move this into `client/gutenberg/extensions`? It aligns with the babel override that's in place there.

Reply:
That's a good idea.

Reply:
I had to remove `import/no-extraneous-dependencies`, as there's no `package.json` in this dir.

Reply:
done in https://github.com/Automattic/wp-calypso/pull/31759/commits/a5b305ed2171cae85165a284fa56e1a456c3dabd

Reply:
Looks like these configs were also in overrides file:

https://github.com/Automattic/wp-calypso/blob/8f4f95a70f5327175c54dddb2fe7b8057ee0593e/.eslintrc.js#L24-L46

...so I removed the config file in https://github.com/Automattic/wp-calypso/pull/31759/commits/eca830cae495502d6e7ed082c55732f567acad50